packages/runtime/src/next/client.ts

```diff
@@ -39,11 +39,40 @@ import {
   Data,
 } from '../state/react-page'
 import { getMakeswiftSiteVersion, MakeswiftSiteVersion } from './preview-mode'
+import { toIterablePaginationResult } from './utils/pagination'
+
+const makeswiftPageResultSchema = z.object({
+  id: z.string(),
+  path: z.string(),
+  title: z.string().nullable(),
+  description: z.string().nullable(),
+  canonicalUrl: z.string().nullable(),
+  socialImageUrl: z.string().nullable(),
+  sitemapPriority: z.number().nullable(),
+  sitemapFrequency: z
+    .enum(['always', 'hourly', 'daily', 'weekly', 'monthly', 'yearly', 'never'])
+    .nullable(),
+  createdAt: z.string(),
+  updatedAt: z.string(),
+  publishedAt: z.string().nullable(),
+  isOnline: z.boolean().nullable(),
+  excludedFromSearch: z.boolean().nullable(),
+  locale: z.string(),
+  localizedVariants: z.array(
+    z.object({
+      locale: z.string(),
+      path: z.string(),
+    }),
+  ),
+})
 
-export type MakeswiftPage = {
-  id: string
-  path: string
-}
+const makeswiftGetPagesResultAPISchema = z.object({
+  data: z.array(makeswiftPageResultSchema),
```

Comment:
Did we have other candidates for the name of this field other than `data`?

Reply:
We considered `results` and `pages`. Ultimately, we went with `data` since it appears to be a standard for paginated APIs (Stripe currently uses `data` in their response: : https://docs.stripe.com/api/pagination)

Reply:
`results` is another common one; Stripe uses `data`: https://docs.stripe.com/api/pagination

Reply:
I've seen `list` as well. I think `data` is good 👍🏻